src/tools/WSWatchers.js

```diff
@@ -0,0 +1,26 @@
+import WS from './WS';
+
+/**
+ * Handles all calls to REST Api
+ */
+class WSWatchers {
+
+    //
+    // WATCHERS Support
+    //
+
+    getWatchersForWorkOrder(woCode, config = {}) {
+        return WS._get(`/watchers/${woCode}`, config);
+    }
+
+    addWatchersToWorkOrder(woCode, users, config = {}) {
+        return WS._put(`/watchers/${woCode}`, users, config);
+    }
+
+    removeWatchersFromWorkOrder(woCode, users, config = {}) {
+        return WS._put(`/watchers/remove/${woCode}`, users, config);
+    }
```

Comment:
Are these new endpoints? If so, this PR should mention that it depends on the backend PR